bootstrap.js

```diff
@@ -97,12 +99,16 @@ function startup(data, reason) {
   Cu.import("resource://tabcenter/verticaltabs.jsm");
   unload(vtInit());
   watchWindows(function(window) {
-    let vt = new VerticalTabs(window);
+    let vt = new VerticalTabs(window, {newPayload, addPingStats});
     unload(vt.unload.bind(vt), window);
   }, "navigator:browser");
+  timer.initWithCallback({notify: () => {
+    sendPing();
+  }}, 24*60*60*1000, Ci.nsITimer.TYPE_REPEATING_SLACK);  // Every 24h…
```

Comment:
So, [this page](https://github.com/bwinton/VerticalTabs/blob/master/docs/metrics.md#data-collection-details) says
> At launch, we will record and send data every 24 hours or on shutdown of the browser, whichever is first.

thus, I’m sending a ping with data every 24 hours…